app/GRZMAK/GRZMAK_interface.h

```diff
@@ -3,6 +3,11 @@
 *
 *
 *History Log:
+<<<<<<< HEAD
+* 2017.12.07 included task3 library
+=======
+>>>>>>> GRZMAK_task_2_set_bit_implementation
```

Comment:
Do not commit such things - it signals not resolved git config (always use `git status` after merging to see which files are conflicted)